CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+### Fixes
+
+- [Pull request #753: Add visual cue for when a dialog is available](https://github.com/alphagov/accessible-autocomplete/pull/753)
```

Comment:
**question** Could this be big enough a change that it warrants a 'New feature' entry with a little more noise?